.gitignore

```diff
@@ -15,3 +15,4 @@
 .rspec_status
 
 /spec.log
+/spec/dummy/.controlplane/controlplane-tmp-*.yml
```

Comment:
Ignore any temporary config files, in case the cleanup in `dummy_app_setup.rb` doesn't work.